README.md

```diff
@@ -91,8 +91,8 @@ can also be found at **https://preview.golem.network/**
 
 ## Useful links:
 
-- Javascript docs: https://docs.golem.network/creators/javascript/quickstart/
-- Dapps docs: https://docs.golem.network/creators/dapps/ 
+- Javascript docs: https://docs.golem.network/docs/creators/javascript/quickstart/
```

Comment:
broken. needs to be https://docs.golem.network/docs/creators/javascript/quickstarts/quickstart